thunderfish/consistentfishes.py

```diff
@@ -1,18 +1,25 @@
+"""
+Functions to compare fishlists created by the harmonicgroups.py module in order to create a create a fishlist with only
+these fishes available in all fishlists.
+
+consistent_fishes(): Compares a list of fishlists and builds a consistent fishlist.
+"""
+
 import numpy as np
 import harmonicgroups as hg
 
 
 def find_consistency(fundamentals, df_th=1.):
     """
-    Compares several lists to find values that are, with a certain threshhold, available in every list.
+    Compares lists of floats to find these values consistent in every list. (with a certain threshold)
```

Comment:
Compares lists of floats and finds consistent values among them. The values may differ according to a defined threshold